tests/css_postcss_plugins/postcss-nesting.css

```diff
@@ -0,0 +1,192 @@
+a {
```

Comment:
Can you switch this file to 2-space indentation so it’s easier to see how the output is changed?